CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+- Breaking change: Remove Google Analytics code (migrated to static)
+
+  As this library has been deprecated the code has been [migrated to static](https://github.com/alphagov/static/pull/1795) for use in GOV.UK Publishing.
+
+  Note that any code that explicitly relies upon the analytics code in toolkit will no longer work.
```

Comment:
I think if we let people know what they should do if they want to continue using this code (copy the code as well?), this'll be good to go.

Reply:
Changed, any better?